src/frontend/web_application/src/components/ContactDetails/components/IdentityDetails/index.jsx

```diff
@@ -1,54 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import Button from '../../../Button';
 import Icon from '../../../Icon';
-import { ItemContent } from '../../../TextList';
 
-import './style.scss';
+const IdentityDetails = ({ identity }) => (
```

Comment:
As discussed with gandians (@ziir, @themouette, ...) I started to use PureComponent instead of function.

It prevents to rewrite the whole component like this.
Just change Component by PureComponent when removing the state manipulations